### ISSUE TITLE
Tradução do termo "top level" para raíz

### ISSUE THREAD

**Issue by rayonnunes** [reactjs/pt-br.react.dev#711]
Verificando o [Glossário](https://github.com/reactjs/pt-br.react.dev/blob/main/GLOSSARY.md) notei que a sugestão para a tradução de _top level_ está como **raiz**, no entanto pode haver alguma confusão como em [Importing and Exporting Components - The root component file](https://react.dev/learn/importing-and-exporting-components#the-root-component-file) onde o termo **raiz** neste caso seria uma tradução adequada para _root_.

Particularmente  acredito que o termo **escopo principal** ou **nível principal** transmite uma ideia mais clara e diferencia do termo _root_. Gostaria de saber o que acham.

| Palavra/Termo original | Sugestão                   |
| --------------------------------- |------------------------------|
| at the top level               | no escopo principal |

**Comment by soutovnc** [comment]
Opa, não sei se ainda está com a mão nisso mas vi em no [GLOSSARY.md](https://github.com/reactjs/pt-br.react.dev/blob/main/GLOSSARY.md) tem uma sugestão para essa tradução. Espero que ajude ;)

**Comment by rayonnunes** [comment]
@soutovnc sim, fiz a observação à partir do glossário mesmo, notei que a tradução atual pode gerar alguma confusão por usar `raiz` tanto como tradução de `root` quanto de `top-level`
